plugins/status/plugin.go

```diff
@@ -214,26 +225,36 @@ func (p *Plugin) oneShot(ctx context.Context) error {
 		req.Metrics = map[string]interface{}{p.metrics.Info().Name: p.metrics.All()}
 	}
 
-	resp, err := p.manager.Client(p.config.Service).
-		WithJSON(req).
-		Do(ctx, "POST", fmt.Sprintf("/status/%v", p.config.PartitionName))
-
-	if err != nil {
-		return errors.Wrap(err, "Status update failed")
+	if p.config.ConsoleLogs {
+		err := p.logUpdate(req)
+		if err != nil {
+			p.logError("Failed to log to console: %v.", err)
+		}
 	}
 
-	defer util.Close(resp)
-
-	switch resp.StatusCode {
-	case http.StatusOK:
-		return nil
-	case http.StatusNotFound:
-		return fmt.Errorf("Status update failed, server replied with not found")
-	case http.StatusUnauthorized:
-		return fmt.Errorf("Status update failed, server replied with not authorized")
-	default:
-		return fmt.Errorf("Status update failed, server replied with HTTP %v", resp.StatusCode)
+	if p.config.Service != "" {
+		resp, err := p.manager.Client(p.config.Service).
+			WithJSON(req).
+			Do(ctx, "POST", fmt.Sprintf("/status/%v", p.config.PartitionName))
+
+		if err != nil {
+			return errors.Wrap(err, "Status update failed")
+		}
+
+		defer util.Close(resp)
+
+		switch resp.StatusCode {
+		case http.StatusOK:
+			return nil
+		case http.StatusNotFound:
+			return fmt.Errorf("status update failed, server replied with not found")
+		case http.StatusUnauthorized:
+			return fmt.Errorf("status update failed, server replied with not authorized")
+		default:
+			return fmt.Errorf("Status update failed, server replied with HTTP %v", resp.StatusCode)
```

Comment:
Nit: this error starts with uppercase S and the others with lowercase s. Let's make this consistent across all plugins.